### DISCUSSION TITLE
Any way to use Outline on Smart TV?

### DISCUSSION THREAD

**Discussion by BossyBigBoss** [outline/outline#7579] [General]
I am looking for a way to use Outline on Smart TVs like Samsung (based on Tizen) or LG (based on WebOS).
All I found is using Shadowsocks, but it is very hard to understand how to set it. 
May be there is exist an easy option?

**Comment by tommoor** [comment]
You're looking for https://github.com/Jigsaw-Code/outline-apps

**Reply by BossyBigBoss** [reply]
It does not help me at all. 
I would be grateful if someone describes "how to" for Smart TVs.
